scripts/install_third_party_libs_test.py

```diff
@@ -53,6 +53,7 @@ def setUp(self):
         def mock_check_call(unused_cmd_tokens, *args, **kwargs):
             self.check_function_calls['check_call_is_called'] = True
             class Ret(python_utils.OBJECT):
+
```

Comment:
not a correct fix, right? I was expecting this to fail on pre-push checks, can you confirm that your pre-push checks are working fine?

Reply:
Note: I see a similar change below.

Reply:
Fixed now.